hawc/apps/mgmt/templates/mgmt/assessment_details.html

```diff
@@ -1,16 +1,105 @@
-      </div>
+<h2 class="d-inline-block">Task assignments</h2>
+<div class="btn-group float-right">
+  <a class="btn btn-primary dropdown-toggle" data-toggle="dropdown">Actions</a>
+  <div class="dropdown-menu dropdown-menu-right">
+    <a class="dropdown-item" href="{% url 'mgmt:user_assessment_assignments' assessment.pk %}">My assigned tasks</a>
   </div>
-  <div id='main' class='mb-3'></div>
-{% endblock %}
+</div>
+
+<!-- filter by study type section not wired up  -->
+<div class="mb-3">
+  <div>
+    <div class="container-fluid filterContainer">
+      <div class="row">
+        <div class="col-md-4">
+          <div class="form-group">
+            <label>Study type filter (optional):</label>
+            <select multiple="" class="form-control" size="4">
+              <option value="bioassay">Animal bioassay</option>
+              <option value="epi">Epidemiology</option>
+              <option value="epi_meta">Epidemiology meta-analysis</option>
+              <option value="in_vitro">In vitro</option>
+              <option value="eco">Ecology</option>
+            </select>
+          </div>
+        </div>
+        <div class="col-md-4">
+          <div class="form-group">
+            <label>Sort studies by:</label>
+            <div class="form-check"><input class="form-check-input" type="radio" id="sortBy-short_citation"
+                name="sortBy" checked="">
+              <label class="form-check-label" for="sortBy-short_citation">Short citation</label>
+            </div>
+            <div class="form-check"><input class="form-check-input" type="radio" id="sortBy-created" name="sortBy">
+              <label class="form-check-label" for="sortBy-created">Creation date</label>
+            </div>
+          </div>
+        </div>
+        <div class="col-md-4">
+          <div class="form-group">
+            <label>Order studies by:</label>
+            <div class="form-check">
+              <input class="form-check-input" type="radio" id="orderBy-asc" name="orderBy" checked="">
+              <label class="form-check-label" for="orderBy-asc">Ascending</label>
+            </div>
+            <div class="form-check">
+              <input class="form-check-input" type="radio" id="orderBy-desc" name="orderBy">
+              <label class="form-check-label" for="orderBy-desc">Descending</label>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
 
-{% block extrajs %}
-  {{ config|json_script:"config" }}
-{% endblock extrajs %}
+    <table class="table table-sm">
+      <colgroup>
+        <col width="20%">
+        <col width="20%">
+        <col width="20%">
+        <col width="20%">
+        <col width="20%">
+      </colgroup>
+      <thead>
+        <tr>
+          <th><span class="popovers" data-title="Study" data-content="Selected study object" data-original-title=""
```

Comment:
do these popovers work? I think they didn't earlier; if they dont you might have to add some JS to turn them on: https://getbootstrap.com/docs/4.6/components/popovers/

Reply:
fixed the popovers. task study type filter still is still a `TODO`. Should I try and approach it (not sure where to start)?